low-level/sensors/impl/platforms/realstor/realstor_logical_volume_sensor.py

```diff
@@ -48,7 +48,12 @@ class RealStorLogicalVolumeSensor(SensorThread, InternalMsgQ):
     SENSOR_NAME = "RealStorLogicalVolumeSensor"
     SENSOR_RESP_TYPE = "enclosure_logical_volume_alert"
     RESOURCE_CATEGORY = "cortx"
-    RESOURCE_TYPE = "enclosure:cortx:logical_volume"
+    RESOURCE_TYPE_LVOL = "enclosure:cortx:logical_volume"
+    RESOURCE_TYPE_DG = "enclosure:cortx:disk_group"
+    SENSOR_DATA_VERSION = '1' # Starting from LDR-R2
```

Comment:
Whats the need really here, for versioning data separately for this sensor ?
Also, version should co-relate with LDR R2 version, would be easier. Let it be 2 even when its being introduced now in R2.

Reply:
If we're okay with keeping two separate files for storing data, we won't need the version field. Removing the version logic and moving disk-group data to another file.

Reply:
yeah better to keep it in most simplistic way, it would help during update also